store/copr/coprocessor.go

```diff
@@ -981,6 +981,8 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *tikv.R
 		logutil.Logger(bo.GetCtx()).Warn("other error",
 			zap.Uint64("txnStartTS", worker.req.StartTs),
 			zap.Uint64("regionID", task.region.GetID()),
+			zap.Uint64("bucketsVer", task.bucketsVer),
```

Comment:
Should log `resp.pbResp.GetLatestBucketsVersion()` too.

Reply:
Good catch.